src/areafix.c

```diff
@@ -416,6 +416,8 @@ char * help(s_link * link)
         w_log(LL_AREAFIX, "%s: HELP sent to %s", af_robot->name, link->name);
         return help;
     }
+    else
+        w_log(LL_ERR, "%s: Help file is not defined", af_robot->name);
```

Comment:
The `fidoconfig` setting is called `HelpFile`. The message should use the name.

Reply:
That's the human response, not the variable name. Shall we use variable name instead?

Reply:
We should give a user as precise information as possible. Since it's the `fidoconfig` setting that is not defined, it is better to mention the setting name. And please put the `wlog(...);` into curly brackets, as it is done in the rest of the code.

Reply:
Done.